caas/kubernetes/provider/k8s.go

```diff
@@ -1781,6 +1789,9 @@ func (k *kubernetesClient) configureService(
 			serviceType = core.ServiceTypeLoadBalancer
 		case caas.ServiceExternal:
 			serviceType = core.ServiceTypeExternalName
+		case caas.ServiceOmit:
```

Comment:
this conflicts with "if hasService" above

Reply:
ah, right, this case is unreachable. removing